modules/controller/index.js

```diff
@@ -70,21 +70,16 @@ Controller.prototype.read = function (opts) {
     opts = {};
   }
   var source = this.source;
-  var type = source.typeName();
   var filters = this._filters.bind(this);
   var relations = this._relations.bind(this);
   // allow a custom responder to receive the payload
   var respond = opts.responder || responder;
   return function (request, response, next) {
-    var validRelations = relations(request).concat(opts.include || []);
     source.read({
       filters: filters(request),
-      relations: validRelations
+      relations: relations(request).concat(opts.include || [])
     }, function (err, data) {
-      var payload = readResponse(err, data, _.extend({}, opts, {
```

Comment:
We shouldn't need it in the future, no. But, we need it now. I'm not quite ready with versioning the API that consumes this codebase. I promised @jugglinmike no more breaking releases that weren't opt-in. We can retire this soon, though.